projects/igniteui-angular/src/lib/core/styles/components/list/_list-theme.scss

```diff
@@ -457,11 +454,11 @@
         color: var-get($theme, 'item-thumbnail-color');
 
         > igx-icon {
-            @include if-ltr() {
+            [dir='ltr'] & {
```

Comment:
This will and doesn't work if the `dir` attribute is not explicitly set, which it is not by default and requiring the users to explicitly be setting direction is a breaking change in terms of behavior, plus we shouldn't make them explicitly set the direction when LTR is used. Additionally, I would prefer to switch to logical properties for padding, instead of using the `padding` shorthand